tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/contentassist/Bug365368Test.java

```diff
@@ -19,7 +19,7 @@
   @Test
   public void testEscapedIdentifierNoPrefix() throws Exception {
     ContentAssistProcessorTestBuilder _newBuilder = this.newBuilder();
-    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\n\t\t\tdef void ^as() {}\n\t\t\tdef void aa() {}\n\t\t\tdef void bb() {}\n\t\t\tdef void m() { this.");
+    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\r\n\t\t\tdef void ^as() {}\r\n\t\t\tdef void aa() {}\r\n\t\t\tdef void bb() {}\r\n\t\t\tdef void m() { this.");
```

Comment:
Hmm the unrelated changes seem to follow a certain pattern. Is the project configuration broken for xtend.ide.tests?